test/e2e/scenario/rum/resources.scenario.ts

```diff
@@ -193,8 +193,8 @@ describe('rum resources', () => {
 
         await flushEvents()
 
-        // Aborted fetch should not be considered as error
-        expect(events.rumErrors.length).toBe(0)
+        // Aborted XHR should be considered as errors for now
```

Comment:
```suggestion
        // Aborted fetch should be considered as errors for now
```
or am I missing something?